daisy/compute/compute.go

```diff
@@ -241,7 +245,17 @@ func shouldRetryWithWait(tripper http.RoundTripper, err error, multiplier int) b
 
 // NewClient creates a new Google Cloud Compute client.
 func NewClient(ctx context.Context, opts ...option.ClientOption) (Client, error) {
-	o := []option.ClientOption{option.WithScopes(compute.ComputeScope, compute.DevstorageReadWriteScope)}
+	// Set these scopes to be align with compute.NewService
+	o := []option.ClientOption{
```

Comment:
We can just drop this and use the deafaults.

Reply:
I followed what "NewService" do. The "NewService" always extends these scopes.
Imagine arg opts gave a narrow scope for any reason, without this extending, it will fail.


// NewService creates a new Service.
func NewService(ctx context.Context, opts ...option.ClientOption) (*Service, error) {
	scopesOption := option.WithScopes(
		"https://www.googleapis.com/auth/cloud-platform",
		"https://www.googleapis.com/auth/compute",
		"https://www.googleapis.com/auth/compute.readonly",
		"https://www.googleapis.com/auth/devstorage.full_control",
		"https://www.googleapis.com/auth/devstorage.read_only",
		"https://www.googleapis.com/auth/devstorage.read_write",
	)
	// NOTE: prepend, so we don't override user-specified scopes.
	opts = append([]option.ClientOption{scopesOption}, opts...)